pipelines/_ingredients/LSSTComCam/DRP.yaml

```diff
@@ -0,0 +1,149 @@
+    class: lsst.analysis.tools.tasks.SourceTableVisitAnalysisTask
+    config:
+      connections.outputName: preSourceTableCore
+      atools.skyFluxVisitStatisticMetric: SkyFluxStatisticMetric
+      atools.skyFluxVisitStatisticMetric.applyContext: VisitContext
+      atools.skySourceSky: SkySourceSkyPlot
+      atools.skySourceFlux: SkySourceHistPlot
+      python: |
+        from lsst.analysis.tools.atools import *
+        from lsst.analysis.tools.contexts import *
+  catalogMatchPreVisit:
+    class: lsst.analysis.tools.tasks.astrometricCatalogMatch.AstrometricCatalogMatchVisitTask
+    config:
+      connections.catalog: "preSourceTable_visit"
+      connections.targetCatalog: "preSourceTable_visit"
+  astrometricRefCatPreSourceVisit:
+    class: lsst.analysis.tools.tasks.refCatSourceAnalysis.RefCatSourceAnalysisTask
+    config:
+      # Only run metrics for analyzing the preSources:
+      connections.data: preSourceTable_visit_uw_stars_20240524_match_astrom
+      atools.astromDiffMetrics: TargetRefCatDeltaMetrics
+      atools.astromDiffMetrics.applyContext: VisitContext
+      python: |
+        from lsst.analysis.tools.atools import *
+        from lsst.analysis.tools.contexts import *
+  matchedVisitCoreWholeSkyPlot:
+    class: lsst.analysis.tools.tasks.WholeSkyAnalysisTask
+    config:
+      connections.inputName: "matchedVisitCore_metricsTable"
+      connections.outputName: "matchedVisitCore_wholeSky"
+      atools.wholeSkyMetric: WholeSkyPlotTool
+      atools.wholeSkyMetric.xLimits: [120.0, 305.0]
+      atools.wholeSkyMetric.yLimits: [-55.0, 10.0]
+      atools.wholeSkyMetric.bands: ["g", "r", "i"]
+      atools.wholeSkyMetric.plotKeys: []
+      atools.wholeSkyMetric.keysWithBand: parameters.matchedVisitCoreKeysWithBand
+      python: |
+        from lsst.analysis.tools.atools import *
+  objectTableCoreWholeSkyPlot:
+    class: lsst.analysis.tools.tasks.WholeSkyAnalysisTask
+    config:
+      atools.wholeSkyMetric: WholeSkyPlotTool
+      atools.wholeSkyMetric.xLimits: [120.0, 305.0]
+      atools.wholeSkyMetric.yLimits: [-55.0, 10.0]
+      atools.wholeSkyMetric.bands: ["g", "r", "i"]
+      atools.wholeSkyMetric.plotKeys: [
+        "wPerpPSFP_wPerp_psfFlux_median",
+        "wPerpPSFP_wPerp_psfFlux_sigmaMAD",
+        "skippedDeblenderMetrics_numSkippedPeaks",
+        "skippedDeblenderMetrics_numSkippedBlends",
+        "skippedDeblenderMetrics_numBlendParentTooBig",
+        "skippedDeblenderMetrics_numBlendTooManyPeaks",
+        "skippedDeblenderMetrics_numBlendTooManyMasked",
+      ]
+      atools.wholeSkyMetric.keysWithBand: parameters.objectTableCoreKeysWithBand
+      python: |
+        from lsst.analysis.tools.atools import *
+  # These don't actually work now:
+  photometricMatchPreVisit:
+    class: lsst.analysis.tools.tasks.photometricCatalogMatch.PhotometricCatalogMatchVisitTask
+    config:
+      extraColumns: ["x", "y", "ap09Flux", "ap09FluxErr"]
+      connections.refCatalog: uw_stars_20240524
```

Comment:
All these refcats will need to change. You can delete any lines with "uw_stars" in them or leave them, but they'll need to be revisited on the next ticket.

Reply:
Ah yes, good point, thanks. I've just done a find & replace on all those to change them to `gaia_dr3_20230707`, because that's at least a little less wrong - I think that's probably right for the astrometry bits, and means that we should then only need to change the photometry parts. Hopefully that's better than leaving them, anyway.